Documentation/Signoffs/Resistance_System_Control.md

```diff
@@ -0,0 +1,123 @@
+|     Bluetooth Module                                                                        |     The HC06 module is an incredibly cheap Bluetooth module.   It should be capable of providing the required range, as it has a maximum   range of about 10 m. In addition to being inexpensive, this module also has   the benefit of being very easily connected; it requires only four pins to be   connected to the external circuit.                                                                                                                                                                                                                                                                                                                     |
+|     Power Connector                                                                         |     The component handles the power input to the circuit,   which is then fed into the linear actuator. As such, it must be rated to   handle slightly over 7 amperes of current. An XT30 connector was selected, as this   type is rated for 15 A which is well over the expected current demand. It   does not allow the user to connect the polarity incorrectly. In addition, the   mating connector is commonly available in a form that lends itself to the   easy fabrication of cables of a custom length without any special equipment.              
+
+To meet the mountability constraint, screw holes have been included in the PCB layout.
+
+To meet the stability constraint, the Arduino microcontroller will be configured as a PID controller. An Arduino library exists to faciliate programming this functionality [1]. Pololu provides a configuration file for the actuator which Team 4 intends to use containing PID constants tuned to control the device in a stable manner [2]. These constants can reasonably be assumed to provide stable performance. The constants given by Pololu (approximated to three significant figures) are as follows: 1.80, 0.0900, and 2.50 for the proportional, integral, and derivative constants, in that order.
+
+The practically-usable actuator length for our design has been determined to be 23 mm. The usable range was determined in the Resistance System signoff, and corresponds to a the range from 2-25 mm away from the flywheel. The 10-bit ADC allows for the full range of the actuator to be divided up into 1024 unique states. The full length of the actuator is 1.97" or 50.04 mm. 23 mm is 45.97 % of 50.04 mm, meaning this portion of the actuator's range corresponds to a 45.97 % of the total number of digital states detectable by the microcontroller; this value is floor[0.4597*1024] = 470, which meets the resolution constraint. This number of unique states corresponds to a spatial (distance) precision of 23/470 = 0.04894 mm or 48.94 micrometers.
```

Comment:
This needs to show that there are 85 states possible at low and high speeds.

Reply:
Done; I have evaluated the full speed range with 0.01 RPM precision. See the matlab program, min_distance_finder.m for detail on the calculations.